src/bundle/Resources/public/js/scripts/admin.location.add.translation.js

```diff
@@ -1,22 +1,28 @@
-(function (global, doc) {
+(function(global, doc) {
     const CLASS_SELECTED = 'ez-translation__label--selected';
     const SELECTOR_WRAPPER = '.ez-translation__language-wrapper';
     const SELECTOR_LABEL = '.ez-translation__label';
     const SELECTOR_MODAL = '.ez-modal';
+    const resetSelection = () => {
+        doc.querySelectorAll(`${SELECTOR_LABEL} .ez-translation__input`).forEach((input) => {
+            input.checked = false;
+        });
+    };
     const updateSelection = (event) => {
         event.preventDefault();
 
         const radio = event.currentTarget.querySelector('input[type="radio"]');
         const checked = !(radio.checked && radio.name === 'add-translation[base_language]');
-        const methodName = checked ? 'setAttribute' : 'removeAttribute';
 
-        [...event.target.closest(SELECTOR_WRAPPER).querySelectorAll(SELECTOR_LABEL)].forEach(label => label.classList.remove(CLASS_SELECTED));
+        [...event.target.closest(SELECTOR_WRAPPER).querySelectorAll(SELECTOR_LABEL)].forEach((label) =>
+            label.classList.remove(CLASS_SELECTED)
+        );
 
         if (checked) {
             event.currentTarget.classList.add(CLASS_SELECTED);
         }
 
-        radio[methodName]('checked', checked);
+        radio.checked = checked;
```

Comment:
If you set the `checked` state this way, then it won't be visible in the HTML attributes. Are you aware of this?

Reply:
It won't be visible in HTML, but CSS `:checked` selector, etc. still work. I also checked default browser behaviour and when you select radio button it does not set HTML attribute. Thus, I don't think that we should do this either.
But, for sure, it was an interesting point from your side. 🙂